CodingDojo/server/src/main/java/com/codenjoy/dojo/services/grpc/handler/UpdateHandler.java

```diff
@@ -40,41 +46,76 @@
 
     private final Map<String, StreamObserver<LeaderboardResponse>> streamObservers;
 
+    private final ExecutorService executorService;
+    private final BlockingQueue<Participant> participants;
+
     public UpdateHandler(GameSaver gameSaver, Registration registration) {
         this.gameSaver = gameSaver;
         this.registration = registration;
         this.streamObservers = new HashMap<>();
+
+        this.executorService = Executors.newSingleThreadExecutor();
+        this.participants = new LinkedBlockingQueue<>(10);
```

Comment:
We should not set capacity for the queue else it will throw exceptions if it is full.